packages/editor/src/components/post-publish-panel/test/__snapshots__/index.js.snap

```diff
@@ -0,0 +1,184 @@
+      aria-expanded={true}
+      icon="no-alt"
+      label="Close panel"
+    />
+  </div>
+  <div
+    className="editor-post-publish-panel__content"
+  >
+    <WithSelect(PostPublishPanelPostpublish) />
+  </div>
+  <div
+    className="editor-post-publish-panel__footer"
+  >
+    <WithInstanceId(CheckboxControl)
+      label="Always show pre-publish checks."
+    />
+  </div>
+</div>
+`;
+
+exports[`PostPublishPanel should render the pre-publish panel if the post is not saving, published or scheduled 1`] = `
+<div
+  className="editor-post-publish-panel"
+>
+  <div
+    className="editor-post-publish-panel__header"
+  >
+    <div
+      className="editor-post-publish-panel__header-publish-button"
+    >
+      <WithSelect(WithDispatch(PostPublishButton))
+        focusOnMount={true}
+        onSubmit={[Function]}
+      />
+      <span
+        className="editor-post-publish-panel__spacer"
+      />
+    </div>
+    <IconButton
+      aria-expanded={true}
+      icon="no-alt"
+      label="Close panel"
+    />
+  </div>
+  <div
+    className="editor-post-publish-panel__content"
+  >
+    <WithSelect(PostPublishPanelPrepublish) />
+  </div>
+  <div
+    className="editor-post-publish-panel__footer"
+  >
+    <WithInstanceId(CheckboxControl)
+      label="Always show pre-publish checks."
+    />
+  </div>
+</div>
+`;
+
+exports[`PostPublishPanel should render the pre-publish panel is post status is scheduled but date is before now 1`] = `
+<div
+  className="editor-post-publish-panel"
+>
+  <div
+    className="editor-post-publish-panel__header"
+  >
+    <div
+      className="editor-post-publish-panel__header-publish-button"
+    >
+      <WithSelect(WithDispatch(PostPublishButton))
+        focusOnMount={true}
+        onSubmit={[Function]}
+      />
+      <span
+        className="editor-post-publish-panel__spacer"
+      />
+    </div>
+    <IconButton
+      aria-expanded={true}
+      icon="no-alt"
+      label="Close panel"
+    />
+  </div>
+  <div
+    className="editor-post-publish-panel__content"
+  >
+    <WithSelect(PostPublishPanelPrepublish) />
+  </div>
+  <div
+    className="editor-post-publish-panel__footer"
+  >
+    <WithInstanceId(CheckboxControl)
+      label="Always show pre-publish checks."
+    />
+  </div>
+</div>
+`;
+
+exports[`PostPublishPanel should render the spinner if the post is being saved 1`] = `
+<div
+  className="editor-post-publish-panel"
+>
+  <div
+    className="editor-post-publish-panel__header"
+  >
+    <div
+      className="editor-post-publish-panel__header-publish-button"
+    >
+      <WithSelect(WithDispatch(PostPublishButton))
+        focusOnMount={true}
+        onSubmit={[Function]}
+      />
+      <span
+        className="editor-post-publish-panel__spacer"
+      />
+    </div>
+    <IconButton
+      aria-expanded={true}
+      icon="no-alt"
+      label="Close panel"
+    />
+  </div>
+  <div
+    className="editor-post-publish-panel__content"
+  >
+    <_default />
```

Comment:
We're only asserting that a function named `_default` is rendered here, which isn't quite right.

This is because of how we define the `Spinner` component:

```js
export default () => <span className="components-spinner" />;
```

It should probably be:

```js
export default function Spinner() {
	return <span className="components-spinner" />;
}
```

Reply:
8a6c0bc